tox.ini

```diff
@@ -5,7 +5,7 @@ isolated_build = true
 [testenv]
 allowlist_externals = coverage
 skip_install = true
-commands = coverage run -m unittest discover -s test
+commands = coverage run -m unittest discover -v
```

Comment:
fix for #26

with `-s START, --start-directory START`, it tested only `./test` directory and not `./integration_test` directory, so I just removed `-s`.

and add `-v` just for more information on test output